efficientdet/keras/train_lib.py

```diff
@@ -183,6 +184,10 @@ def get_optimizer(params):
     optimizer = tf.keras.optimizers.Adam(learning_rate)
   else:
     raise ValueError('optimizers should be adam or sgd')
+  if params['moving_average_decay']:
+    # Only work on tfa-nightly
+    import tensorflow_addons as tfa
+    optimizer = tfa.optimizers.MovingAverage(optimizer, dynamic_decay=True)
```

Comment:
Very nice! But I did not find "dynamic_decay" option on the doc: https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/MovingAverage

Looks like you should also set average_decay=params['moving_average_decay']?

Reply:
Only work with addons nightly
https://github.com/tensorflow/addons/blob/a0bfe3f35f310bdfee53078bdee15e10af342f53/tensorflow_addons/optimizers/moving_average.py#L52